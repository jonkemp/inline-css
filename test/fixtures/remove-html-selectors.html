<html><link rel="stylesheet" href="remove-html-selectors.css"/><body>
<div id="myId" class="myClass">Test</div>
<div id="myId" class="myClass">Test</div></body></html>
