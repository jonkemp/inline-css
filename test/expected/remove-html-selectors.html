<html><head></head><body>
<div style="color: red; display: inline-block; font-size: 12px; width: 100%;">Test</div>
<div style="color: red; display: inline-block; font-size: 12px; width: 100%;">Test</div>
</body></html>
